broker-cli/utils/validations.js

```diff
@@ -65,9 +73,9 @@ function isMarketName (str) {
       return str.toUpperCase()
     }
 
-    throw new Error()
+    throw new Error('[empty error to trigger the catch]')
   } catch (e) {
-    throw new Error('Market Name format is incorrect')
+    throw new ValidationError(`Market Name should be specified with '--market <marketName>', where <marketName> is two symbols separated by a '/' e.g. 'BTC/LTC'`)
```

Comment:
Is there a name for this format? (maybe official terminology from investopedia)

Reply:
Reason why I ask is because of the explanation in this error

Reply:
I'm not aware of any official designation from traditional finance, but usually hear it referred to as 'trading pair'. `base` and `quote` should be fairly ubiquitous across the industry, so this could be rephrased similar to:

`-- market <marketName>, where <marketName> is the trading pair designated as <baseToken>/<quoteToken>`

That said, I think the the error message as stated would be enough for the user to figure out the correct format.